packages/core/build/build.tokens.ts

```diff
@@ -168,24 +170,30 @@ function flattenTokens(theme: any) {
   return flatten(theme);
 }
 
-function convertCSSValue(token: Token, base = 20, fallback = true) {
+function convertCSSValue(token: Token, fallback = true) {
   let value = token.value;
 
   if (token.alias instanceof Token) {
     value = `var(--cds-${camelCaseToKebab(token.alias.name)}${
-      fallback ? `, ${convertCSSValue(token.alias, 20, fallback)}` : ''
+      fallback ? `, ${convertCSSValue(token.alias, fallback)}` : ''
     })`;
-  } else if (token.config.static) {
+  } else if (token.config.raw) {
     value = token.value;
+  } else if (token.config.scale) {
+    value = `calc(${token.value} * var(--cds-${camelCaseToKebab(token.config.scale.name)}))`;
   } else if (typeof token.value === 'number') {
-    value = `calc((${token.value} / var(--cds-global-base, ${base})) * 1rem)`;
+    value = `calc(${token.value} * (1rem / var(--cds-global-base)))`;
```

Comment:
Adding support for new token value types. The `raw` option enables a token to be defined with no conversions such as px to rem. This is helpful for tokens for calcs or absolute values like in media queries. The `scale` allows a token to define a multiplier against another number type token. This is helpful for creating "knobs" to scale a size up or down easily.